engine/apps/user_management/models/user.py

```diff
@@ -183,6 +183,12 @@ class Meta:
     def __str__(self):
         return f"{self.pk}: {self.username}"
 
+    @property
+    def available_teams(self):
```

Comment:
property that returns teams where user has access to based on membership, team is shared
if user is admin they are able to see all the org teams